src/twisted/logger/test/test_format.py

```diff
@@ -95,6 +95,14 @@ def where(self) -> str:
             "hello world", self.format("hello {what.where()}", what=World())
         )
 
+    def test_formatTypes(self) -> None:
```

Comment:
I think that is best to keep `formatEvent` in the name, to highlight that this is a test for the `t.logger.formatEvent` public API.
 
Can was also have a test for `twisted.logger.eventAsText` which is the failing API in the initial bug report?

```suggestion
    def test_formatEventTypes(self) -> None:
```

Reply:
I changed `formatEvent` test to `eventAsText` one. Not sure if we should add all higher-level APIs tests then.